app/assets/v2/js/grants/fund.js

```diff
@@ -84,7 +84,7 @@ $(document).ready(function() {
           }).on('transactionHash', function(transactionHash) {
             $('#sub_new_approve_tx_id').val(transactionHash);
             const linkURL = etherscan_tx_url(transactionHash);
-            var token_address = $('#js-token').length ? $('#js-token').val() : $("#sub_token_address").val();
+            var token_address = $('#js-token').length ? $('#js-token').val() : $('#sub_token_address').val();
```

Comment:
Why was this changed? Just to clean up the style?

Reply:
I ran eslint:fix and it pointed it out